docs/pages/pageheader.js

```diff
@@ -397,43 +591,98 @@ function Example() {
       </Box>
     </Box>
   </Box>
-</Flex>
+</Box>
         `}
           shaded
         />
       </MainSection>
 
       <MainSection
         name="Localization"
-        description={`Be sure to localize the \`title\`, \`subtext\` and actions within PageHeader.`}
+        description={`Be sure to localize the \`badge\`, \`dropdownAccessibilityLabel\`, \`helperIconButton\`, \`helperLink\`, \`title\`, \`subtext\`, as well as any \`primaryAction\`, \`secondaryAction\` and \`item\` components used within PageHeader.
+
+Be brief with text in all components to account for languages with longer words.`}
       >
         <MainSection.Card
           shaded
           cardSize="lg"
           defaultCode={`
-<Box width="100%">
-  <PageHeader
-    title="Langer Seitentitel, der in einigen Sprachen möglicherweise abgeschnitten wird"
-    subtext="2.131 Produkte"
-    primaryAction={{
-      component: <Button color="red" size="lg" text="Schnell erstellen" />,
-      dropdownItems: [
-        <Dropdown.Item
-          option={{ value: '', label: 'Schnell erstellenSchnell erstellen' }}
-          onSelect={() => {}}
-        />,
-      ],
-    }}
-    dropdownAccessibilityLabel="Zusatzoptionen"
-  />
-</Box>
-  `}
+<PageHeader
+  title="Anzeigenübersicht"
+  subtext='5 aktive Kampagnen.'
+  helperLink={{
+    text: "Mehr erfahren.",
+    accessibilityLabel: "Erfahren Sie mehr auf Pinterest.com",
+    href: "http://www.pinterest.com",
+    onClick: () => {} }}
+  items={[
+    <Datapoint
+      size="md"
+      title="Impressionen"
+      value="$1.25M"
+      trend={{ value: 30, accessibilityLabel: 'Aufwärtstrend' }}
+    />,
+  ]}
+  primaryAction={{
+    component: <Button color="red" size="lg" text="Fördern" />,
+    dropdownItems: [
+      <Dropdown.Item option={{ value: 'Fördern', label: 'Fördern' }} onSelect={() => {}} />,
+    ],
+  }}
+  secondaryAction={{
+    component: <Button size="lg" text="Analysen anzeigen" />,
+    dropdownItems: [
+      <Dropdown.Link
+        option={{ value: 'Analysen anzeigen', label: 'Analysen anzeigen' }}
+        href="https://pinterest.com"
+      />,
+    ],
+  }}
+  dropdownAccessibilityLabel="Mehr Optionen"
+/>
+    `}
         />
       </MainSection>
-
       <MainSection name="Variants">
         <MainSection.Subsection
-          description={`PageHeader supports an optional \`primaryAction\`. It can be a [Button](/button), a [Link](/link) or an [IconButton](/iconbutton) with a [Tooltip](/tooltip) and optional [Dropdown](/dropdown). Any Buttons or IconButtons should be \`size="lg"\`.`}
+          description={`PageHeader's \`title\` is the main part of the component as it represents the page's main heading (it will always be a level 1 heading). It can be complemented with three additional elements: a thumbnail (left) and a badge and/or a helperIcon (right). Don't forget to localized its content.`}
```

Comment:
help Icon instead of helperIcon? or put helperIcon in back-ticks maybe?

Reply:
DONE